src/main/java/ua/lviv/iot/parkingServer/logic/VehicleService.kt

```diff
@@ -0,0 +1,50 @@
+package ua.lviv.iot.parkingServer.logic
+
+import org.springframework.stereotype.Service
+import ua.lviv.iot.parkingServer.datastorage.VehicleFileStore
+import ua.lviv.iot.parkingServer.model.Vehicle
+import javax.annotation.PostConstruct
+import javax.annotation.PreDestroy
+
+@Service
+class VehicleService(private val vehicleFileStore: VehicleFileStore) {
+    private val vehicles: MutableMap<Long, Vehicle> = HashMap()
+    private var index: Long = 0L
```

Comment:
First of all, please rename to `lastId`(or something like that)

Second. This implementation is not thread safe. This service will be broken as soon as two threads call `addVehicle` simultaneously. It's better to use [AtomicLong ](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/AtomicLong.html) here